docs/docs/noir/standard_library/bigint.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Big Integers
+description: How to use big integers from Noir standard library
+keywords:
+  [
+    Big Integer,
+    Noir programming language,
+    Noir libraries,
+  ]
+---
+
+The BigInt module in the standard library exposes some class of integers which do not fit (well) into a Noir native field. It implements modulo arithmetic, modulo a 'big' prime number.
+Currently only 6 class of integers (i.e 'big' prime numbers) are available throughout the module, namely:
```

Comment:
```suggestion

:::note

The module can currently be considered as `Field`s mimicked on a set of curves with corresponding set modulo sizes. [More work](https://github.com/noir-lang/noir/issues/510) is needed to achieve arbitrarily sized big integers.

:::

Currently 6 classes of integers (i.e 'big' prime numbers) are available in the module, namely:
```

Reply:
I am not fan of "mimicked on a set of curves". I'd rather say something like this:
`The module can currently be considered as `Field`s with modulo sizes used by a set of useful elliptic curves.`

Reply:
👍 Suggestion updated, see if that's better?